bom/pom.xml

```diff
@@ -41,7 +41,7 @@
     <netty.version>4.1.8.Final</netty.version>
     <objenesis.version>2.4</objenesis.version>
     <proton.version>0.18.0</proton.version>
-    <qpid-dispatch.version>0.7.0</qpid-dispatch.version>
+    <qpid-dispatch.version>0.8.0</qpid-dispatch.version>
```

Comment:
I wasn't able to build the eclipsehono/qpid-dispatch image using Dispatch Router 0.8.0.
```
Step 1/3 : FROM fedora:latest
 ---> 0047cca29c6f
Step 2/3 : RUN dnf -y install libuuid openssl                    cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-sql cyrus-sasl-gssapi cyrus-sasl-md5                    qpid-dispatch-router-0.8.0 qpid-dispatch-tools-0.8.0 &&     dnf -y update &&     dnf clean all
 ---> Running in a1e21a9f4187
Last metadata expiration check: 0:00:10 ago on Thu May 11 08:17:56 2017.
Package libuuid-2.28.2-2.fc25.x86_64 is already installed, skipping.
Package cyrus-sasl-lib-2.1.26-26.2.fc24.x86_64 is already installed, skipping.
No package qpid-dispatch-router-0.8.0 available.
No package qpid-dispatch-tools-0.8.0 available.
Error: Unable to find a match.
The command '/bin/sh -c dnf -y install libuuid openssl                    cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-sql cyrus-sasl-gssapi cyrus-sasl-md5                    qpid-dispatch-router-0.8.0 qpid-dispatch-tools-0.8.0 &&     dnf -y update &&     dnf clean all' returned a non-zero code: 1
```

Seems like we will need to wait a little more for the 0.8.0 Fedora packages being released ...